### ISSUE TITLE
Npc: Deathguard Oliver movement bug

### ISSUE THREAD

**Issue by rec0d** [MillipedeAS/onyx-wow#16]
Npc: Deathguard Oliver (ID: 1737)  flying for 2 sec when walking up the hill movement bug  [location] .tele deathknell

**Comment by MillipedeAS** [comment]
Though this is easily fixed, with some new waypoints.

**Comment by rec0d** [comment]
Yes its easy, but you disapeared on skype. Need you to start the server again
